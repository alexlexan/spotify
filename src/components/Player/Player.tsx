import React from "react";
import Footer from "../Footer/Footer";
import "./Player.sass";
import Sidebar from "../Sidebar/Sidebar";
import Body from "../Body/Body";
import { ErrorBoundary } from "react-error-boundary";
import ErrorFallback from "../ErrorFallback";

const Player: React.FC = () => {
  return (
    <div className="player">
      <div className="player__body">
        <Sidebar />
        <ErrorBoundary FallbackComponent={ErrorFallback}>
          <Body />
        </ErrorBoundary>
      </div>
      <Footer />
    </div>
  );
};

export default Player;
