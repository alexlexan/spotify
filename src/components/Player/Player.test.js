import React from "react";
import Player from "./Player.tsx";

describe("tests Player component", () => {
  it("render Player component", () => {
    const component = shallow(<Player />);
    expect(component).toMatchSnapshot();
  });
});
