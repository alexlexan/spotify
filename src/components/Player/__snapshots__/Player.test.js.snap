// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests Player component render Player component 1`] = `
<div
  className="player"
>
  <div
    className="player__body"
  >
    <Sidebar />
    <ErrorBoundary
      FallbackComponent={[Function]}
    >
      <Body />
    </ErrorBoundary>
  </div>
  <Footer />
</div>
`;
